packages/universal-components/src/RangeDatePicker/DayItemArrow.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+
+import * as React from 'react';
+import { Animated, Easing, View, Platform } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import { type StylePropType, StyleSheet } from '../PlatformStyleSheet';
+import { designTokens } from '../DesignTokens';
+
+type Props = {|
+  +style?: StylePropType,
+|};
+
+type State = {|
+  scale: Animated.Value,
+|};
+
+const scaleAnimationDuration =
+  parseFloat(defaultTokens.durationSlow) * 2 * 1000;
```

Comment:
Does `durationSlow` from `defaultTokens` still makes sense here if we modify it that much?
What about adding new value into universal tokens?

Reply:
True 😄 done